resources.management.tf

```diff
@@ -137,3 +136,71 @@ resource "azurerm_log_analytics_linked_service" "management" {
   ]
 
 }
+
+resource "azurerm_user_assigned_identity" "management" {
+  for_each = local.azurerm_user_assigned_identity_management
+
+  provider = azurerm.management
+  # Mandatory resource attributes
+  name                = each.value.template.name
+  location            = each.value.template.location
+  resource_group_name = each.value.template.resource_group_name
+
+  # Optional resource attributes
+  tags = each.value.template.tags
+
+  # Set explicit dependency on Resource Group deployment
+  depends_on = [
+    azurerm_resource_group.management,
+  ]
+}
+
+resource "azurerm_monitor_data_collection_rule" "management" {
```

Comment:
Should we support `description` and `tags` on this resource?

Reply:
Updated thanks!